### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by theaswanson**
This PR updates the packages to their latest versions. Most notably, TypeScript gets upgraded from v4 to v5, and Storybook goes from v6 to v8.

I was having issues building this project and running Storybook on Windows, and these updates appear to fix that.

If there's anything you'd like me to change, please let me know!

### CODE REVIEW

File: .prettierrc
Comment (nachovigilante) [inline .prettierrc:1]: Please don't change the default formatting rules

File: dist/cjs/components/Digit.d.ts
Comment (nachovigilante) [inline dist/cjs/components/Digit.d.ts:1]: I left `/dist` folder by accident, it shouldn't be included in the repository. I'm sorry! :(

**Review by nachovigilante — CHANGES_REQUESTED** [review]
I like the idea of updating the packages. Revert the changes on formatting and I'll accept it. Don't worry about the `/dist` directory, I'll remove it from the repo later. Thanks for investing time on this!

**Comment by theaswanson** [comment]
@nachovigilante I've reverted the formatting and removed the changes to `dist`. Let me know what you think!

**Review by nachovigilante — APPROVED** [review]
Thanks for taking the time!

**Pull request merged**